tests/common/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/FlowsScopePassingSpanSubscriberTests.java

```diff
@@ -56,6 +56,7 @@
 public abstract class FlowsScopePassingSpanSubscriberTests {
 
 	static final String HOOK_KEY = "org.springframework.cloud.sleuth.autoconfig.instrument.reactor.TraceReactorAutoConfiguration.TraceReactorConfiguration";
+	static final String LIFTER_NAME = "org.springframework.cloud.sleuth.instrument.reactor.ReactorHooksHelper.ScopePassingLifter";
```

Comment:
maybe mark this as a copy of the original in ReactorHooksHelper, if the constant cannot be directly accessed by this piece of code

Reply:
added note